server/sql_scripts/getStateDetails.sql

```diff
@@ -8,8 +8,13 @@ CREATE PROCEDURE `getUserStateDetails` (
   )  READS SQL DATA
 
     BEGIN
+<<<<<<< HEAD
```

Comment:
This is a merge conflict and should be resolved before submitting.

Reply:
That said, Im not sure which is correct. I figure the bottom one is. Either way, it'll get worked out before it gets fully merged into dev (but will be really confusing if the head and other merge issues are left in and then compounded when merged to dev).